tools/docs/manager.ts

```diff
@@ -0,0 +1,86 @@
+import type { RenovateConfig } from '../../lib/config/types';
+import { getManagers } from '../../lib/manager';
+import { readFile, updateFile } from '../utils';
+import { getDisplayName, getNameWithUrl, replaceContent } from './utils';
+
+function getTitle(manager: string, displayName: string): string {
+  if (manager === 'regex') {
+    return `Custom Manager Support using Regex`;
+  }
+  return `Automated Dependency Updates for ${displayName}`;
+}
+
+function getManagerLink(manager: string): string {
+  return `[\`${manager}\`](${manager}/)`;
+}
+
+export async function generateManagers(dist: string): Promise<void> {
+  const managers = getManagers();
+  const allLanguages: Record<string, string[]> = {};
+  for (const [manager, definition] of managers) {
+    const language = definition.language || 'other';
+    allLanguages[language] = allLanguages[language] || [];
+    allLanguages[language].push(manager);
+    const { defaultConfig } = definition;
+    const { fileMatch } = defaultConfig as RenovateConfig;
+    const displayName = getDisplayName(manager, definition);
+    let md = `---
+title: ${getTitle(manager, displayName)}
+sidebar_label: ${displayName}
+---
+`;
```

Comment:
This part of the code looks weird, seems we're missing some tabs/spaces?

Reply:
copy'n'paste, won't like to change this, as this would cause docs to change. we can fix it later 😉